api/src/db/migrations/v122_remove_welcome_and_welcomeupandrunning_sidecards.test.ts

```diff
@@ -0,0 +1,27 @@
+import {
+  generateV121Business,
+  generateV121Preferences,
+  generateV121UserData,
+} from "./v121_add_nonprofit_formation_fields";
+import { migrate_v121_to_v122 } from "./v122_remove_welcome_and_welcomeupandrunning_sidecards";
+
+describe("migrate_v121_to_v122", () => {
+  it("removes the welcome and welcome-up-and-running from sidebarCards", () => {
+    const id = "biz-1";
+    const v121Preferences = generateV121Preferences({
+      visibleSidebarCards: ["welcome", "welcome-up-and-running", "other-card"],
+    });
+    const v121Business = generateV121Business({ id: id, preferences: v121Preferences });
+    const v121UserData = generateV121UserData({ businesses: { "biz-1": v121Business } });
+
+    expect(v121UserData.businesses[id].preferences.visibleSidebarCards).toEqual([
+      "welcome",
+      "welcome-up-and-running",
+      "other-card",
+    ]);
+    expect(v121UserData.version).toEqual(121);
```

Comment:
What is the purpose of these 2 expectations? It seems like we're testing whether the test setup functions worked correctly...?

Reply:
Just testing that those cards are in there in 121 but I can take them out..